create guides section and getting started file (#1)
otfbm/docs: guides_getting_started.md

# Getting Started

> If you're running the `!map` alias for the first time look no further. This guide will teach you how to set up a map, customize it, place characters as tokens, and move them. Check the [Advanced Guides]() if you've already got a handle on the `!map` alias.

Author: ```suggestion
If you're running the `!map` alias for the first time look no further. This guide will teach you how to set up a map, customize it, place characters as tokens, and move them. Check out our [Advanced Guides]() if you've already got a handle on the `!map` alias.
```

> **Disclaimer**
This guide assumes you know the basic `!init` functions of Avrae: how to start an encounter, add monsters, add combatants, and generally run a combat. If you don't you can head over to [Avrae's documentation](https://avrae.readthedocs.io/en/latest/) to find out how.

## 1. Starting initiative and setting up the map

> First up, we start the initiative by typing the command:

Author: ```suggestion
First up, we start enter initiative by typing the command:
```

Reviewer: ```suggestion
First up, we enter initiative by typing the command:
```

Reviewer: Did you mean this instead, @digitalsadhu ?

Author: yes!

```
!init begin
```

> Once that is done, we need to add a character to the initiative order to whom we're going to attach the map. We recommend adding a dedicated character for this, instead of a player or a monster, to prevent the map from vanishing when a player exits combat, or a monster is killed.

Author: ```suggestion
Once that is done, we need to add a character to the initiative order onto whom we're going to attach the map. We recommend adding a dedicated character for this, instead of a player or a monster, to prevent the map from vanishing when a player exits combat, or a monster is killed.
```

So our character is going to be called `DM`, but it can be called anything you want, `Map`, `Lair`, `Whatever`. And we’re going to set its initiative at 20 so it can also be used for Lair Actions, if needed:

```
!i add 0 DM -p 20
```

Next, we're going to attach the map itself to our `DM` character, to bring it up:

```
!map -mapattach DM
```

Without attaching a map to a character, every time you run the `!map` command the previous changes or settings are lost and the map reverts to its default. This way everything is saved.

## 2. Customizing the map

Now that you have a map going, you're probably thinking that the default 10x10 grid isn't big enough. And you're right, we need to go bigger! So, let's try making it a 26x14 grid instead:

```
!map -mapsize 26x14
```

Great, but it’s still a bland white background. Let's spice things up by adding a background image:

```
!map -mapbg https://i.imgur.com/WnTUKcJ.jpg 
```

> Looking better. You can add any background you like provided the image lines up with a 40 by 40 pixel grid or it will be cropped.

Author: ```suggestion
Looking better. You can add any background you like with the following caveats:
* Images larger than 1mb may be resized or lose resolution
* The default gridsize is 40px x 40px, if the map you use is not 40x40, you may need to adjust the gridsize with the `@c` option and offset the background image with the `@o` option.
```

Reviewer: What about this? Because I had something similar to your suggestion and then edited.
```suggestion
Looking better. You can add any background you like with the following caveats:
- images larger than 1MB may be resized or lose resolution unit they're reduced to that file size;
- since the default grid size is 40px by 40px the background image might not line up. If that's the case, check out our [Advanced Guides]() to find out how to change the grid size or offset the background image.
```

> Sweet. So now we have the map anchored to the initiative through a specific character, set to a good size, and with a custom background. That’s a lot of commands to run separately. Luckily, you can use all those arguments on the same command instead:

Author: ```suggestion
Sweet. So now we have the map anchored to the initiative order through a specific combatant, set to a good size, and with a custom background. That’s a lot of commands to run separately. Luckily, you can use all those arguments on the same command instead:
```
